modules/distr-tracing-tempo-install-tempomonolithic-web-console.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-install-tempomonolithic-web-console_{context}"]
+= Installing a TempoMonolithic instance by using the web console
+
+:FeatureName: The TempoMonolithic instance
+include::snippets/technology-preview.adoc[]
+
+You can install a TempoMonolithic instance from the *Administrator* view of the web console.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as a cluster administrator with the `cluster-admin` role.
+
+* For {product-dedicated}, you must be logged in using an account with the `dedicated-admin` role.
+
+.Procedure
+
+. Create a project of your choice for the *TempoMonolithic* instance that you will create in a subsequent step: go to *Home* -> *Projects* -> *Create Project*.
```

Comment:
```suggestion
. Create a project of your choice for the *TempoMonolithic* instance that you create in a subsequent step by clicking *Home* -> *Projects* -> *Create Project*.
```

Reply:
Here is how I rephrased this step:

_. Go to *Home* -> *Projects* -> *Create Project* to create a project of your choice for the *TempoMonolithic* instance that you will create in a subsequent step._